src/dbus/client/thread_api_dbus.hpp

```diff
@@ -38,7 +38,6 @@
 
 #include <dbus/dbus.h>
 
-#include "common/code_utils.hpp"
```

Comment:
We need `OTBR_UNUSED_VARIABLE` from this header file. If we remove this inclusion then `EmptyFree` needs to be moved into the cpp file.

Reply:
I removed `aData` from `EmptyFree`. How do you think?